client/js/app.js

```diff
@@ -3,8 +3,7 @@ angular
     'ui.router',
     'lbServices'
   ])
-  .config(['$stateProvider', '$urlRouterProvider', function($stateProvider,
-      $urlRouterProvider) {
+  .config(['$stateProvider', '$urlRouterProvider', function ($stateProvider, $urlRouterProvider) {
```

Comment:
>... function ($stateProvider, $urlRouterProvider) {

You've added a space in front front of `function` and didn't break the line past 80 chars. Please fix this too.